File: src/main/java/net/atos/client/zgw/zrc/model/zaakobjecten/ObjectPand.java
/*
 * SPDX-FileCopyrightText: 2022 Atos, 2023-2024 Lifely
 * SPDX-License-Identifier: EUPL-1.2+
 */
package net.atos.client.zgw.zrc.model.zaakobjecten;

/**
 * ObjectPand
 */
public class ObjectPand extends ObjectBAGObject {

  /**
   * Constructor for JSONB deserialization
   */
  public ObjectPand() {}

  /**
   * Constructor with required attributes
   */
  public ObjectPand(final String identificatie) {
    super(identificatie);
  }
}

File: src/main/kotlin/nl/info/client/zgw/zrc/model/zaakobjecten/ObjectPand.kt
/*
 * SPDX-FileCopyrightText: 2023, 2026 INFO.nl
 * SPDX-License-Identifier: EUPL-1.2+
 */
package nl.info.client.zgw.zrc.model.zaakobjecten

import nl.info.zac.util.NoArgConstructor

/**
 * ObjectPand
 */
@NoArgConstructor
data class ObjectPand(override var identificatie: String? = null) : ObjectBagObject(identificatie)
